cmd/nbe/tmpl/head.html

```diff
@@ -6,10 +6,10 @@
 <link rel="stylesheet" href="/main.css">
 
 <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-1YB2EC4YMG"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-9QV780SHZ5"></script>
 <script>
   window.dataLayer = window.dataLayer || [];
   function gtag(){dataLayer.push(arguments);}
   gtag('js', new Date());
-  gtag('config', 'G-1YB2EC4YMG');
+  gtag('config', 'G-9QV780SHZ5');
```

Comment:
Remove this tracking ID, since the current one is correct.

Reply:
Thank you. I made a commit